### ISSUE TITLE
[feature request] support authorization for non SSH connection

### ISSUE THREAD

**Issue by paulvi** [Kanatoko/MonjaDB#21]
I had to stop using plugin as our development team added authorization to our MongoDB instances

![monjiadb-no-auth](https://f.cloud.github.com/assets/1614482/1472751/22c9e072-4609-11e3-9a8f-3803ccd90b30.png)
. 

There is no fields for user, password, so I believe it is impossible as of 1.0.16

**Comment by Kanatoko** [comment]
Yes, currently authorization is not supported.

**Comment by paulvi** [comment]
Do you have plans to support it (this year) ?

**Comment by Kanatoko** [comment]
I don't think I can implement this feature in this year.

**Comment by paulvi** [comment]
OK

**Comment by paulvi** [comment]
I dived into code

```
public class MConnectAction
```

has 

```
public void executeFunction()
throws Exception
{
checkExistingConnection();
mongo = new Mongo( host, port );
db = mongo.getDB( dbName );
}
```

Mongo class from `mongo-2.7.2.jar` has no methods or parameters to set auth (user, password) 

@Kanatoko Could you give any hints.

Then I looked into 30a0c0956ce45dd47d5fa155fc7cc24cd4a527c9 (why is it called auth ?), search by auth and found `MAuthCommand`
Could you comment what was your intent?
Why not to `db.authenticate( username, passwd.toCharArray() )` on default `admin` db by default?

![auth-settings](https://f.cloud.github.com/assets/1614482/1498493/84228fde-4836-11e3-8a85-459b9622056f.png)

**Bottom line: How would you advise  (how do you see) to do authentication?**

**Comment by Kanatoko** [comment]
If you write code, I think you should ignore my old implementation. 
Because I don't really understand how MongoDB auth system works.

**Comment by paulvi** [comment]
I see, so before was first try.
